core/src/test/java/com/mapzen/android/graphics/MapzenMapTest.java

```diff
@@ -293,10 +293,10 @@
     verify(overlayManager).onFling(1f, 2f, 3f, 4f);
   }
 
-  @Test public void setRotateResponder_shouldInvokeMapController() {
+  @Test public void setRotateResponder_shouldNotInvokeMapController() {
```

Comment:
Great! Is there something that I need to do? Or you will incorporate all these changes?